build/terraform/e2e/module.tf

```diff
@@ -62,3 +62,16 @@ resource "helm_release" "consul" {
     value = "ClusterIP"
   }
 }
+
+resource "google_compute_firewall" "tcp" {
```

Comment:
Seem to be running into the same problem that the TCP ports are not open, so not sure why this is not working

Reply:
This will need to be some variation of `gke-` because reasons -- but also makes me wonder if we should allow you to specific the protocols (UDP, TCP, etc) supported when creating the cluster and accompanying firewall?